app/javascript/components/zone-form/index.jsx

```diff
@@ -0,0 +1,52 @@
+import React, { useEffect, useState } from 'react';
+
+import MiqFormRenderer from '@@ddf';
+import createSchema from './zone-form.schema';
+import miqRedirectBack from "../../helpers/miq-redirect-back";
+import { http } from '../../http_api';
+
+const ZoneForm = ({recordId}) => {
+    const [{initialValues, isLoading}, setState] = useState({ isLoading: !!recordId });
+
+    const cancelUrl = recordId ? `/ops/zone_edit/${recordId}?button=cancel` : '/ops/zone_edit/?button=cancel';
+
+    const onSubmit = ({href, id, created_on, updated_on, actions, authentications, ...data}) => {
+        miqSparkleOn();
+        const message = __('Zone was saved');
+        const submitUrl = recordId ? `/ops/zone_edit/?button=save&id=${recordId}&name=${data.name}` : `/ops/zone_edit/?button=add&name=${data.name}`;
+
+        const request = recordId ? API.post(`/api/zones/${recordId}`, {action: "edit", resource: {...data, authentications: { authentications }}}) : 
```

Comment:
This should allow us to do PATCH/PUT: https://github.com/ManageIQ/manageiq-api/pull/897

Reply:
For the `authentications` I created this issue: https://github.com/ManageIQ/manageiq-api/issues/898